woof-code/packages-templates/gtk3_FIXUPHACK

```diff
@@ -13,7 +13,7 @@ if [ -d usr/share/icons/Adwaita ]; then
 	rm -rf [0-9]*x* cursor* scalable-*
 	find scalable -type f | while read ICON; do
 		case $ICON in
-		*/pan-*) ;;
+		*/pan-*|*/send-to-symbolic.svg|*/open-menu-symbolic.svg|*/document-edit-symbolic.svg) ;;
```

Comment:
Aren't you just preserving the symbolic links and not the icons which the symbolic links point to?

Reply:
Maybe, we'll know soon :)

Reply:
Works great!

![image](https://user-images.githubusercontent.com/1471149/209783348-81322997-5012-4cdf-b0f2-b3ca4f52b939.png)

Reply:
That theme is the default theme? Seems completely changed!

Reply:
Puppy Standard

![image](https://user-images.githubusercontent.com/1471149/209783954-3a5ec408-c661-4f2d-8ff3-e7ca336886dd.png)

Some GTK+ 3/4 applications use mostly symbolic icons, so they don't follow the chosen icon theme and use those flat icons instead. Blueman is a good example.

Reply:
And only a tiny fraction of Adwaita is present,

![image](https://user-images.githubusercontent.com/1471149/209784269-e8d3b8b6-2a64-4f98-8932-6a0a38b3ea98.png)

Reply:
Should be a great PR, but since we have developed icons (2 are in progress as you know, 1 is resolved), doing this, if I understand correctly, may allow evince to use Adwaita icons instead of puppy icons. Don't know if it will really use Adwaita icons even if those are present in default theme used by user.

Reply:
I prefer the solution proposed by @01micko and won't be offended if we close this PR.

Reply:
I have tested this on my jammy64 by reinstalling whole adwaita theme. Evince does not seem to be respecting PMaterial symlinks. Rather, when I moved those icons from `icon-name.svg` to `icon-name-symbolic.svg`, it seemed to be working. We could actually merge this, just that we'll have to move symlinks to `*-symbolic.svg` for only those two symbolic icons. Still don't know if this solution will work for `document-edit.svg` as well.